core/domain/stats_services_test.py

```diff
@@ -493,6 +512,66 @@ def test_get_top_unresolved_answers(self):
                 'eid', 'sid'), {})
 
+    def test_unresolved_answers_count_for_single_exploration(self):
+        exp_1 = self._create_and_update_fake_exploration('eid1')
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                ['eid1']), {})
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid1', 1, exp_1.init_state_name, self.DEFAULT_RULESPEC_STR, 'a1')
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                ['eid1']), {'eid1': 1})
+
+    def test_unresolved_answers_count_for_multiple_explorations(self):
+        exp_1 = self._create_and_update_fake_exploration('eid1')
+        exp_2 = self._create_and_update_fake_exploration('eid2')
+        exp_3 = self._create_and_update_fake_exploration('eid3')
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                ['eid1', 'eid2', 'eid3']), {})
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid1', 1, exp_1.init_state_name, self.DEFAULT_RULESPEC_STR, 'a1')
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid2', 1, exp_2.init_state_name, self.DEFAULT_RULESPEC_STR, 'a3')
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid2', 1, exp_2.init_state_name, self.DEFAULT_RULESPEC_STR, 'a2')
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid3', 1, exp_3.init_state_name, self.DEFAULT_RULESPEC_STR, 'a2')
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                ['eid1', 'eid2', 'eid3']), {'eid1': 1, 'eid2': 2, 'eid3': 1})
+
+    def test_unresolved_answers_count_when_answers_marked_as_resolved(self):
+        exp_1 = self._create_and_update_fake_exploration('eid1')
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                ['eid1']), {})
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid1', 1, exp_1.init_state_name, self.DEFAULT_RULESPEC_STR, 'a1')
+        event_services.AnswerSubmissionEventHandler.record(
+            'eid1', 1, exp_1.init_state_name, self.DEFAULT_RULESPEC_STR, 'a2')
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                ['eid1']), {'eid1': 2})
+
+        event_services.DefaultRuleAnswerResolutionEventHandler.record(
+            'eid1', exp_1.init_state_name, ['a1'])
+        self.assertEquals(
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
```

Comment:
My rationale for that is that this function, ideally, should handle all types of unresolved answers (this can only be done when the answer migration is completed, and might be out of scope for this project). Perhaps you can add a note in the docstring of the function instead, noting that it currently only deals with the default rule.